### ISSUE TITLE
Fix VQGAN notebook training loop

### ISSUE THREAD

**Issue by mahalrs** [mahalrs/newsgen#14]
Since current VQGAN model is using NN module and not Lightning module, the training loop should handle calls such as optimizer.step, loss.backward(), optimizer.zero_grad(), etc.
